core/registration.c

```diff
@@ -61,6 +61,7 @@
 #include <stdio.h>
 
 #define MAX_LOCATION_LENGTH 10      // strlen("/rd/65534") + 1
+#define MAX_BLOCK1_SIZE 1024
```

Comment:
Would like to see:
#ifndef MAX_BLOCK1_SIZE
#define MAX_BLOCK1_SIZE 1024
#endif
so that it can be overridden for a specific project to select a smaller size.